src/vs/platform/terminal/common/terminal.ts

```diff
@@ -799,6 +799,15 @@ export interface ITerminalContributions {
 	profiles?: ITerminalProfileContribution[];
 }
 
+export interface ITerminalQuickFixContribution {
+	id: string;
+	commandLineMatcher: string | RegExp;
+	outputMatcher: ITerminalOutputMatcher;
+	exitStatus?: boolean;
```

Comment:
think this should be exitCode for more granularity and can have a way to indicate any that failed (maybe -1?)

Reply:
ok yeah will stick w this for now